_events/2019-03-10-first_public_workshop.md

```diff
@@ -12,8 +13,103 @@ location:
 supporters:
   - denbi
   - mozilla
+photos:
+  name: Bérénice Batut
+  license: CC-BY-SA
+  url: https://www.flickr.com/photos/134305289@N03/shares/AmKS19
 ---
 
-Our first public workshop for the [BeerDEcoded project]({% link _projects/beerdecoded.md %}) will be held on the 31st of March in Freiburg.
+On the 31st of March, we hold our first public workshop for the [BeerDEcoded project]({% link _projects/beerdecoded.md %}) in Freiburg. Some students and external people participated to this event: 16 peoples in total, including us.
 
-You want to particpate? Please send us an email at [{{ site.email|replace:'@','[at]' }}](mailto:{{ site.email }}) or join the [mailing list]({{ site.google_group }}).
+{% include event-images.html images="
+  /images/events/2019-03-10-first_public_workshop/DSC_9661.jpg"
+%}
+
+We started the day with short introduction to beer, yeast, DNA, but also lab safety. We organized after some games around pipetting and centrifuge to make the participants more comfortable with both.
```

Comment:
We started the day with a short introduction to beer, yeast, DNA and of course lab safety. Then all participants took part in some fun games on pipetting and centrifuging to make them more comfortable with both.